powerquery-docs/connectors/smartsheet.md

```diff
@@ -15,18 +15,54 @@ ms.author: dougklo
 | ---- | ----------- |
 | Release State | General Availability |
 | Products | Power BI (Semantic models)<br/>Power BI (Dataflows)<br/>Fabric (Dataflow Gen2) |
```

Comment:
it should read "Fabric Dataflow Gen2" and "Power BI Dataflow"

Reply:
Updated to the correct Product data.